R/headers_list.R

```diff
@@ -21,9 +21,12 @@
 
   HEADERS_LIST[["metrics_results"]] <- c(
     "maxlog10Concentration",
+    "maxlog10Concentration_sd",
```

Comment:
`_sd` is a shortcut from..?

Reply:
It's a shortcut from the standard deviation in this case. When we are averaging biological replicates the "default metrics" are in fact means and additional ones complement the default ones with the standard deviation values.

Reply:
So `x_sd_avg_sd` is standard deviation of standard deviation for averaged values? 🤔

Reply:
Exactly.